src/styles/blocks/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  
+  background-color: $base-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+
+  background-image: url(../images/logo.png);
+}
+
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+    justify-content: center;
+    margin: 0;
+  }
+
+  &__item {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+
+    font-weight: 500;
+    font-size: 12px;
+    text-transform: uppercase;
+    color: $main-accent;
+    text-decoration: none;
+    line-height: 60px;
+  }
+
+  &__link:hover {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point